index.html

```diff
@@ -69,5 +77,296 @@
+            <div class="right-block-2">
+              <p class="title-post-2">Profesional Art Printing Data</p>
+              <p class="sup-t">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+            </div>
+            <div class="gap"></div>
+            <ul class="tags-2">
+              <li class="five one1">
+                <a class="html" href="html.org">html</a>
+              </li>
+              <li class="five two2">
+                <a class="bootstrap" href="bootstrap.org">bootstrap</a>
+              </li>
+              <li class="five three3">
+                <a class="ruby" href="ruby.org">ruby</a>
+              </li>
+            </ul>
+            <div class="action action3 btnh">
+              <a class="button " href="#">see project</a>
+            </div>
+          </div>
+          <div class="card-works">
+            <div class="right-block-2">
+              <p class="title-post-2">Profesional Art Printing Data</p>
+              <p class="sup-t">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+            </div>
+            <div class="gap"></div>
+            <ul class="tags-2">
+              <li class="five one1">
+                <a class="html" href="html.org">html</a>
+              </li>
+              <li class="five two2">
+                <a class="bootstrap" href="bootstrap.org">bootstrap</a>
+              </li>
+              <li class="five three3">
+                <a class="ruby" href="ruby.org">ruby</a>
+              </li>
+            </ul>
+            <div class="action action4 btnh">
+              <a class="button" href="#">see project</a>
+            </div>
+          </div>
+          <div class="card-works">
+            <div class="right-block-2">
+              <p class="title-post-2">Profesional Art Printing Data</p>
+              <p class="sup-t">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+            </div>
+            <div class="gap"></div>
+            <ul class="tags-2">
+              <li class="five one1">
+                <a class="html" href="html.org">html</a>
+              </li>
+              <li class="five two2">
+                <a class="bootstrap" href="bootstrap.org">bootstrap</a>
+              </li>
+              <li class="five three3">
+                <a class="ruby" href="ruby.org">ruby</a>
+              </li>
+            </ul>
+            <div class="action action5 btnh">
+              <a class="button " href="#">see project</a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="about">
+      <h1 class="primery">About me</h1>

```

Comment:
- [ ] Kindly change this `h1` tag to any of the `hx (h2-h6)` tags. It is the convention and best practice to use only one `h1` tag on a webpage. Kindly change it 👍